cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -171,11 +172,11 @@ def validate_size(self, value):
 class QueueRootVolumeSchema(BaseSchema):
     """Represent the RootVolume schema for the queue."""
 
-    size = fields.Int(metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
-    encrypted = fields.Bool(metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
-    volume_type = fields.Str(metadata={"update_policy": UpdatePolicy.COMPUTE_FLEET_STOP})
-    iops = fields.Int(metadata={"update_policy": UpdatePolicy.SUPPORTED})
-    throughput = fields.Int(metadata={"update_policy": UpdatePolicy.SUPPORTED})
+    size = fields.Int(metadata={"update_policy": UpdatePolicy.QUEUE_UPDATE_STRATEGY})
+    encrypted = fields.Bool(metadata={"update_policy": UpdatePolicy.QUEUE_UPDATE_STRATEGY})
+    volume_type = fields.Str(metadata={"update_policy": UpdatePolicy.QUEUE_UPDATE_STRATEGY})
+    iops = fields.Int(metadata={"update_policy": UpdatePolicy.QUEUE_UPDATE_STRATEGY})
```

Comment:
Can we keep SUPPORTED for iops and throughput?

Reply:
no because iops and throughput are set into the Launch Template